porcupine/plugins/run/dialog.py

```diff
@@ -106,7 +106,15 @@ def __init__(self, file_path: Path, project_path: Path, suggestions: list[histor
         )
 
         sub_text = "\n".join("{%s} = %s" % pair for pair in substitutions.items())
-        ttk.Label(content_frame, text=f"Substitutions:\n{sub_text}\n").pack(fill="x")
+        sub_textbox = textutils.create_passive_text_widget(
+            content_frame, height=len(substitutions) + 2
+        )
+        sub_textbox.pack(fill="x")
+        sub_textbox.config(state="normal")
+        sub_textbox.insert("1.0", f"Substitutions:\n{sub_text}\n")
+        sub_textbox.tag_configure("indent", lmargin1=15)
+        sub_textbox.tag_add("indent", "2.0", "end")
```

Comment:
It might be easier to use a separate label for the initial `"Substitutions:"`, since that part doesn't need to be copy/pasteable.